<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WareedLogin</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton:hover{
        background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(183, 44, 24, 219), stop:1 rgba(93, 39, 24, 226));
}

QPushButton:pressed{
        padding-left:5px;
        padding-top:5px;
        background-color:rgba(150, 123, 111, 255);
}

QPushButton{
        background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(124, 44, 24, 219), stop:1 rgba(93, 39, 24, 226));
        color:rgba(255, 255, 255, 210);
        border-radius:5px;
}

QLineEdit{
background-color:rgba(0, 0, 0, 0);
border:none;
border-bottom:2px solid rgba(124, 44, 24, 219);
color:rgba(0, 0, 0, 240);
padding-bottom:7px;

}

</string>
  </property>
  <widget class="QPushButton" name="pushButtonConnect">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>440</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>CONNECT</string>
   </property>
  </widget>
  <widget class="QLabel" name="LablelTitleWareed">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgba(255,255,255,210)</string>
   </property>
   <property name="text">
    <string>Wareed </string>
   </property>
  </widget>
  <widget class="QLabel" name="LableHideYP">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgba(255,255,255,210)</string>
   </property>
   <property name="text">
    <string>Hide Your Payload...</string>
   </property>
  </widget>
  <widget class="QLabel" name="LableVersion">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>360</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgba(255,255,255,210)</string>
   </property>
   <property name="text">
    <string>1.0v</string>
   </property>
  </widget>
  <widget class="QLabel" name="BackGroundWhite">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>0</y>
     <width>291</width>
     <height>521</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(255,255,255,255);
border-radius:0px;
border-bottom-right-radius:50px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="WareedBackGround">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>281</width>
     <height>521</height>
    </rect>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="styleSheet">
    <string notr="true">
border-radius:0px;
border-top-left-radius:50px;
background-color:rgba(0,0,0,80);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>80</y>
     <width>231</width>
     <height>341</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="lineEditUserName">
      <property name="placeholderText">
       <string>User Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditPassword">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>Password</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditIPAddress">
      <property name="placeholderText">
       <string>IPAddress</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdirPort">
      <property name="placeholderText">
       <string>Port</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="LableMadeby">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>490</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Made By Faisal-p27</string>
   </property>
  </widget>
  <widget class="QLabel" name="LablelTitleLogin">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>30</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonQuit">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>0</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>x</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>310</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>60</y>
     <width>271</width>
     <height>271</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Image/WareedIcon.png);</string>
   </property>
  </widget>
  <zorder>WareedBackGround</zorder>
  <zorder>BackGroundWhite</zorder>
  <zorder>pushButtonConnect</zorder>
  <zorder>frame</zorder>
  <zorder>LableMadeby</zorder>
  <zorder>LablelTitleWareed</zorder>
  <zorder>LableVersion</zorder>
  <zorder>LablelTitleLogin</zorder>
  <zorder>pushButtonQuit</zorder>
  <zorder>widget</zorder>
  <zorder>widget_2</zorder>
  <zorder>LableHideYP</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
